distance_until_mission_finished: 5
number_of_laps: 1
time_to_another_lap: 20
distance_to_orange_cone: 4 #if a orange cone is at maximum at this distance this may count as a lap or mission finished
distance_to_last_path_point: 5.0 #distance to the current position and the position of the end of the track, as identified by SLAM
offset_from_start_x: 10 #distance in X axis from the starting position of the mission
offset_from_start_y: 0 #distance in Y axis from the starting position of the mission
maximum_distance_from_offset_position: 5 #radial distance form the offset position where the car is supposed to stop
